2.4.0/ubuntu/14.04/Dockerfile

```diff
@@ -1,16 +1,22 @@
 FROM ubuntu:14.04
 MAINTAINER Peter Ericson <pdericson@gmail.com>
 
-RUN apt-get update && apt-get install --no-install-recommends -y ca-certificates curl && rm -rf /var/lib/apt/lists/*
+ENV MESOS_VERSION=0.28.1-2.0.20.ubuntu1404
+ENV CHRONOS_VERSION=2.4.0-0.1.20151007110204.ubuntu1404
 
 # https://github.com/Yelp/dumb-init
-RUN curl -fLsS -o /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.0.2/dumb-init_1.0.2_amd64 && chmod +x /usr/local/bin/dumb-init
-
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv E56151BF && \
-echo deb http://repos.mesosphere.io/ubuntu trusty main > /etc/apt/sources.list.d/mesosphere.list && \
-apt-get update && \
-apt-get install --no-install-recommends -y chronos=2.4.0-0.1.20151007110204.ubuntu1404 mesos=0.28.1-2.0.20.ubuntu1404 && \
-rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y curl && \
+    curl -fLsS -o /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.0.2/dumb-init_1.0.2_amd64 && \
+    echo '22426dab7b3553459619638e18d2ab9061ddc866  dumb-init' > /tmp/dumb-init.asc && \
+    cd /usr/local/bin && shasum -a1 -c /tmp/dumb-init.asc && cd - && \
+    chmod +x /usr/local/bin/dumb-init && \
+    # Install chronos
```

Comment:
Hi @adamdecaf 

https://travis-ci.org/mesoscloud/chronos/builds/160264994#L261

This is fine from a Dockerfile point of view but if you were to try something similar in a shell the comment would break the command, e.g.

```sh
$ ls a \
>    # b
ls: a: No such file or directory
```